c10/util/complex_math.h

```diff
@@ -2,16 +2,16 @@ namespace std {
 
 // Exponential functions
 
-#if defined(CUDA_VERSION) && (CUDA_VERSION < 10000)
```

Comment:
Instead of removing the condition, could you bump up the version to `CUDA_VERSION < 10200`? This would be helpful for people to know when this workaround can be removed. 10.2 is well tested on CI, and should work.

Reply:
Done

Reply:
@xuhdev I didn't see your commit for this? GitHub is broken today, you might need push again?

Reply:
Pushed again.